src/Docker/Docker.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace DockerHostManager\Docker;
+
+use Docker\Docker as DockerBase;
+
+class Docker extends DockerBase
+{
+    /**
+     * @param string   $entrypoint
+     * @param callable $callback
+     */
+    public function listenEvents($entrypoint, callable $callback)
+    {
+        $stream = stream_socket_client($entrypoint, $errno, $errstr, 30);
```

Comment:
Yes... But when I want to read the stream with `read()`, I have no result. With `read(1024)`, I have no result until stream has 1024 caracters.
With `read(1)` in a loop, it's working... but very dirty!